src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,72 @@
+package duke.task;
+
+import java.time.LocalDate;
+
+import duke.DukeException;
+import duke.util.Parser;
+
+
+/**
+ * Task with a deadline.
+ */
+public class Deadline extends Task {
```

Comment:
Hi, again, not a coding standard violation but I was wondering if adding Task behind the respective classes for tasks will be better? I feel it allows readers to easily navigate to the type of task they want to see. Also, Event could be misleading as it can represent things like button onClick events etc

```suggestion
public class DeadlineTask extends Task {
```